charts/tails-server/README.md

```diff
@@ -0,0 +1,98 @@
+# Tails Server Helm Chart
+
+A simple, production‑ready Helm chart to deploy the Indy Tails Server.
+
+- App: <https://github.com/bcgov/indy-tails-server>
+- Kubernetes: >= 1.23 (uses autoscaling/v2)
+
+## TL;DR
+
+```bash
+# From repo root
+helm upgrade --install tails-server charts/tails-server \
+  --namespace tails --create-namespace
+```
+
+## Prerequisites
+
+- Kubernetes >= 1.23
+- Metrics server (if enabling HPA)
+- A default StorageClass (if enabling persistence without existingClaim)
+
+## Configuration
+
+Values below are the most commonly tuned. See `values.yaml` for the full list.
+
+| Parameter | Default | Description |
+|---|---|---|
+| replicaCount | 1 | Desired replicas when HPA disabled |
+| autoscaling.enabled | false | Enable HorizontalPodAutoscaler |
+| autoscaling.minReplicas | 1 | HPA min replicas |
+| autoscaling.maxReplicas | 4 | HPA max replicas |
+| autoscaling.targetCPUUtilizationPercentage | 80 | HPA CPU target |
+| image.repository | ghcr.io/bcgov/tails-server | Image repository |
+| image.tag | chart appVersion | Image tag |
+| service.type | ClusterIP | Service type |
+| service.port | 6543 | Service port (TCP) |
+| service.annotations | {} | Extra Service annotations |
+| ingress.enabled | false | Create Ingress (OpenShift will auto‑create Route) |
+| persistence.enabled | false | Enable persistent storage for tails files |
```

Comment:
I think this should set to true by default, I don't think there is a scenario where the tails server would not want storage to be provisioned for/with it.

Reply:
updated.